twilio_Sms_send.py

```diff
@@ -23,5 +23,5 @@ def send_twil_msg(content):
     print(message.sid)
 
 
-send_twil_msg("Hello")
+send_twil_msg("HELLO")
```

Comment:
Any specific reason to change to caps ?

Reply:
no

Reply:
Please contribute to my readme file on this repository, That will be really helpful :)